web/modules/custom/joinup_core/joinup_core.deploy.php

```diff
@@ -16,10 +16,57 @@
 
 use Drupal\asset_distribution\Entity\DownloadEvent;
 
+/**
+ * Restore the field policy domain node field into the new topic field.
+ */
+function joinup_core_deploy_0107000(&$sandbox) {
```

Comment:
Normally we should add new functions after existing ones. Also this lacks strict typing for $sandbox and function return.

Reply:
I know and I understand this. This is the standard procedure. However, the problem comes from the fact that I am renaming the topic table in the last step of post updates. If there was to be another deploy hook to run before mine and attempt to update an entity, it would break the deployment due to a missing field table. That is why I wanted my post update to run last, and the deploy to run first. I will add a comment in the hook.